RPerformanceTracking/Private/_RPTTracker.m

```diff
@@ -123,6 +124,28 @@ - (void)updateURL:(NSURL *)url trackingIdentifier:(uint_fast64_t)trackingIdentif
     }
 }
 
+- (void)sendResponseHeader:(NSDictionary *)responseHeader trackingIdentifier:(uint_fast64_t)trackingIdentifier
+{
+    NSString *sourceURL = responseHeader[@"link"];
+    if (!sourceURL.length) {
+        return;
+    }
+    _RPTMeasurement *measurement = [_ringBuffer measurementWithTrackingIdentifier:trackingIdentifier];
+    NSTimeInterval startTime = measurement.startTime * 1000;
+    NSTimeInterval responseEnd = [NSDate.date timeIntervalSince1970] * 1000;
+    NSTimeInterval duration = responseEnd - startTime;
+    NSString *cdn = responseHeader[@"X-CDN-Served-From"];
+    NSMutableDictionary *dataEntry = [NSMutableDictionary dictionary];
+    dataEntry[@"startTime"] = @(startTime);
+    dataEntry[@"responseEnd"] = @(responseEnd);
+    dataEntry[@"duration"] = @(duration);
+    dataEntry[@"name"] = sourceURL;
+    dataEntry[@"cdn"] = cdn;
```

Comment:
Won't this crash if cdn is nil? (inserting nil into a dictionary).

As it is optional I think we should only set it if it is >0 length, like
`if (cdn.length) dataEntry[@"cdn"] = cdn;`